### PR TITLE
[Doc] translate  notes/changelog.md, contribution_guide.md, faq.md

### PR DESCRIPTION
**Pull request by wangbo-zhao**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here and update the documentation.

## Checklist

**Before PR**:

- [ ] Pre-commit or other linting tools are used to fix the potential lint issues.
- [ ] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/zh_cn/notes/changelog.md [hunk 1]
```diff
@@ -1,246 +1,372 @@
-# Changelog
+# 更新日志
 
 ## MMSelfSup
 
+### v1.0.0rc2 (12/10/2022)
+
+`master` 仍然是 0.x 版本，我们将会 checkout 一个新的 `1.x` 用来发布 1.x 版本。 未来我们会同时维护两个版本。
+
+我们简要的列出主要的改变。 请参考[migration guide](../migration.md)来查看细节和迁移指引
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/changelog.md]:
```suggestion
我们简要的列出主要的改变。 请参考[迁移文档](../migration.md)来查看细节和迁移指引
```

File: docs/zh_cn/notes/changelog.md [hunk 2]
```diff
@@ -1,246 +1,372 @@
...
+
+#### 亮点
+
+- 全量支持 `MAE`, `SimMIM`, `MoCoV3`.
+
+#### 新特性
+
+- 全量支持 `MAE` ([#483](https://github.com/open-mmlab/mmselfsup/pull/483))
+  -全量支持 `SimMIM` ([#487](https://github.com/open-mmlab/mmselfsup/pull/487))
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/changelog.md]:
```suggestion
- 全量支持 `SimMIM` ([#487](https://github.com/open-mmlab/mmselfsup/pull/487))
```

File: docs/zh_cn/notes/changelog.md [hunk 3]
```diff
@@ -1,246 +1,372 @@
...
+- 优化 contribution guide ([#492](https://github.com/open-mmlab/mmselfsup/pull/492))
+- 更新 convention ([#475](https://github.com/open-mmlab/mmselfsup/pull/475))
+
+### v1.0.0rc1 (01/09/2022)
+
+我们很高兴宣布发布 MMSelfSup v1.0.0rc1。
+We are excited to announce the release of MMSelfSup v1.0.0rc1. MMSelfSup v1.0.0rc1 是 MMSelfSup 1.x 的第一个版本，是 OpenMMLab 2.0 项目中的一部分。
+`master` 仍然是 0.x 版本，我们将会 checkout 一个新的 `1.x` 用来发布 1.x 版本。 未来我们会同时维护两个版本。
+
+我们简要的列出主要的改变。 请参考[migration guide](../migration.md)来查看细节和迁移指引
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/changelog.md]:
```suggestion
我们简要的列出主要的改变。 请参考[迁移文档](../migration.md)来查看细节和迁移指引
```

File: docs/zh_cn/notes/contribution_guides.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+git clone git@github.com:open-mmlab/mmselfsup.git
+cd mmselfsup
+git checkout dev-1.x
+git checkout -b xxxx # xxxx is the name of new branch
+```
+
+3. 根据后面提到的代码风格编辑相关的文件
+4. 使用 **pre-commit hook** 来检查和格式化您的修改。
+5. 提交您的修改。
+6. 创建一个 PR。
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
6. 创建一个 PR，往 dev-1.x 合入。
```
Add this description of EN version simultaneously

File: docs/zh_cn/notes/faq.md [hunk 5]
```diff
@@ -1,13 +1,14 @@
 # FAQ
 
-We list some common troubles faced by many users and their corresponding solutions here. Feel free to enrich the list if you find any frequent issues and have ways to help others to solve them. If the contents here do not cover your issue, please create an issue using the [provided templates](https://github.com/open-mmlab/mmselfsup/tree/master/.github/ISSUE_TEMPLATE) and make sure you fill in all required information in the template.
+我们列出来一些用户常见的问题，并将他们的解决方案列出。 您可以将一些您发现的常见的问题添加进列表中，来帮助其他用户解决问题。 如果这里面的内容没有覆盖您的问题，请按照 [provided templates](https://github.com/open-mmlab/mmselfsup/tree/master/.github/ISSUE_TEMPLATE) 创建一个 issue，并确保您在模板中填写了所有要求的信息。
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/faq.md]:
```suggestion
我们列出来一些用户常见的问题，并将他们的解决方案列出。 您可以将一些您发现的常见的问题添加进列表中，来帮助其他用户解决问题。 如果这里面的内容没有覆盖您的问题，请按照 [模板](https://github.com/open-mmlab/mmselfsup/tree/master/.github/ISSUE_TEMPLATE) 创建一个 issue，并确保您在模板中填写了所有要求的信息。
```

File: docs/zh_cn/notes/faq.md [hunk 6]
```diff
@@ -20,5 +21,18 @@ Compatible MMCV, MMClassification, MMDetection and MMSegmentation versions are s
 
 **Note:**
 
-- You need to run `pip uninstall mmcv` first if you have mmcv installed. If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
-- If you still have version problem, please create an issue and provide your package versions.
+- MMDetection and MMSegmentation 是可选的。
+- 如果您仍然有版本错误，请创建一个issue并提供您的包的版本信息。
+
+## DeepCluster on A100 GPU
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/faq.md]:
```suggestion
## DeepCluster 在 A100 GPU
```

File: docs/zh_cn/notes/faq.md [hunk 7]
```diff
@@ -20,5 +21,18 @@ Compatible MMCV, MMClassification, MMDetection and MMSegmentation versions are s
 
 **Note:**
 
-- You need to run `pip uninstall mmcv` first if you have mmcv installed. If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
-- If you still have version problem, please create an issue and provide your package versions.
+- MMDetection and MMSegmentation 是可选的。
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/faq.md]:
```suggestion
- MMDetection 和 MMSegmentation 是可选的。
```

File: docs/zh_cn/notes/faq.md [hunk 8]
```diff
@@ -1,13 +1,14 @@
...
+  - [DeepCluster on A100 GPU](#deepcluster-on-a100-gpu)
 
-## Installation
+## 安装
 
-Compatible MMCV, MMClassification, MMDetection and MMSegmentation versions are shown below. Please install the correct version of them to avoid installation issues.
+MMCV, MMClassification, MMDetection and MMSegmentation 的版本兼容性如下所示。 请安装正确的版本来避免安装问题。
 
 | MMSelfSup version |    MMCV version     |  MMClassification version   | MMSegmentation version | MMDetection version |
 | :---------------: | :-----------------: | :-------------------------: | :--------------------: | :-----------------: |
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/faq.md]: add 1.x version dependency

File: docs/zh_cn/notes/changelog.md (hunk 1 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/changelog.md]: done

File: docs/zh_cn/notes/changelog.md (hunk 2 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/changelog.md]: done

File: docs/zh_cn/notes/changelog.md (hunk 3 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/changelog.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 4 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

File: docs/zh_cn/notes/faq.md (hunk 5 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/faq.md]: done

File: docs/zh_cn/notes/faq.md (hunk 7 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/faq.md]: done

File: docs/zh_cn/notes/faq.md (hunk 6 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/faq.md]: done

File: docs/zh_cn/notes/faq.md (hunk 8 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/faq.md]: done

File: docs/zh_cn/notes/contribution_guides.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+- 修复 bug 或者错误
+- 添加或翻译文档为另外一种语言
+- 添加新特性或者组件
+
+## 流程
+
+我们推荐潜在的贡献者使用下面的流程来添加贡献：
+
+1. Fork 并 pull 最新的MMSelfSup仓库，并根据 [get_started](../get_started.md) 来配置环境。
+2. Checkout 一个新的分支 (\*\*不要使用 aster/dev branch 提交 PR)
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
2. Checkout 一个新的分支 (\*\*不要使用 master/dev branch 提交 PR)
```

File: docs/zh_cn/notes/contribution_guides.md [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+git clone git@github.com:open-mmlab/mmselfsup.git
+cd mmselfsup
+git checkout dev-1.x
+git checkout -b xxxx # xxxx is the name of new branch
+```
+
+3. 根据后面提到的代码风格编辑相关的文件
+4. 使用 **pre-commit hook** 来检查和格式化您的修改。
+5. 提交您的修改。
+6. 创建一个 PR，并往dev-1.x合入。
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
6. 创建一个 PR，并往 dev-1.x 合入。
```

File: docs/zh_cn/notes/contribution_guides.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+- [codespell](https://github.com/codespell-project/codespell): 一个修改错误拼写的Python工具。
+- [mdformat](https://github.com/executablebooks/mdformat): Mdformat 是一个可选的 Markdorn 的格式化工具，可以让 Markdown 文件中的格式保持一致。
+- [docformatter](https://github.com/myint/docformatter): 一个格式化 docstring 的工具.
+
+配置 yapf 和 isort 放在在 [setup.cfg](./setup.cfg)中。
+
+我们使用 [pre-commit hook](https://pre-commit.com/) 来检查和格式化`flake8`, `yapf`, `isort`, `trailing whitespaces`, `markdown files`，自动修改`end-of-files`, `double-quoted-strings`, `python-encoding-pragma`, `mixed-line-ending`, sorts `requirments.txt` 在每一次提交。
+为 pre-commit hook 的配置储存在 [.pre-commit-config](./.pre-commit-config.yaml)。
+
+在您 clone 残酷之后，您需要安装并初始化 pre-commit hook。
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
在您 clone 代码库之后，您需要安装并初始化 pre-commit hook。
```

File: docs/zh_cn/notes/contribution_guides.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+## 代码风格
+
+### Python
+
+我们采用 [PEP8](https://www.python.org/dev/peps/pep-0008/) 作为我们的代码风格。
+我们采用一下工具来进行 linting 和 formatting：
+
+- [flake8](https://github.com/PyCQA/flake8): 一个linter工具装饰器.
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
- [flake8](https://github.com/PyCQA/flake8): 一个 linter 工具装饰器.
```

File: docs/zh_cn/notes/faq.md [hunk 13]
```diff
@@ -1,24 +1,40 @@
...
 **Note:**
 
-- You need to run `pip uninstall mmcv` first if you have mmcv installed. If mmcv and mmcv-full are both installed, there will be `ModuleNotFoundError`.
-- If you still have version problem, please create an issue and provide your package versions.
+- MMDetection 和 MMSegmentation 是可选的。
+- 如果您仍然有版本错误，请创建一个issue并提供您的包的版本信息。
+
+## DeepCluster 在 A100 GPU
+
+如果您想尝试 [DeepCluster](https://github.com/open-mmlab/mmselfsup/blob/master/configs/selfsup/deepcluster/README.md) 在 A100 GPU上，使用 pip 安装 `faiss` 将会引发错误，
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/faq.md]:
```suggestion
如果您想尝试 [DeepCluster](https://github.com/open-mmlab/mmselfsup/blob/master/configs/selfsup/deepcluster/README.md) 在 A100 GPU 上，使用 pip 安装 `faiss` 将会引发错误，
```

Comment (wangbo-zhao) [reply docs/zh_cn/notes/faq.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 12 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 11 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 10 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 9 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

**Review by fangyixiao18 — APPROVED** [review]

File: docs/zh_cn/notes/contribution_guides.md [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+- 修复 bug 或者错误
+- 添加或翻译文档为另外一种语言
+- 添加新特性或者组件
+
+## 流程
+
+我们推荐潜在的贡献者使用下面的流程来添加贡献：
+
+1. Fork 并 pull 最新的MMSelfSup仓库，并根据 [get_started](../get_started.md) 来配置环境。
+2. Checkout 一个新的分支 (\*\*不要使用 master/dev branch 提交 PR)
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
2. Checkout 一个新的分支 (**不要使用 master/dev branch 提交 PR**)
```

File: docs/zh_cn/notes/contribution_guides.md [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+
+- 修复 bug 或者错误
+- 添加或翻译文档为另外一种语言
+- 添加新特性或者组件
+
+## 流程
+
+我们推荐潜在的贡献者使用下面的流程来添加贡献：
+
+1. Fork 并 pull 最新的MMSelfSup仓库，并根据 [get_started](../get_started.md) 来配置环境。
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
1. Fork 并 pull 最新的 MMSelfSup 仓库，并根据 [get_started](../get_started.md) 来配置环境。
```

File: docs/zh_cn/notes/contribution_guides.md [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+### Python
+
+我们采用 [PEP8](https://www.python.org/dev/peps/pep-0008/) 作为我们的代码风格。
+我们采用一下工具来进行 linting 和 formatting：
+
+- [flake8](https://github.com/PyCQA/flake8): 一个 linter 工具装饰器.
+- [isort](https://github.com/timothycrosley/isort): 一个为 Python 导入排序的工具。
+- [yapf](https://github.com/google/yapf): 一个为 Python 文件格式化的工具。
+- [codespell](https://github.com/codespell-project/codespell): 一个修改错误拼写的Python工具。
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
- [codespell](https://github.com/codespell-project/codespell): 一个修改错误拼写的 Python 工具。
```

File: docs/zh_cn/notes/contribution_guides.md [hunk 17]
```diff
@@ -0,0 +1,78 @@
+# 向MMSelfSup贡献代码
+
+- [向MMSelfSup贡献代码](#向MMSelfSup贡献代码)
+  - [流程](#流程)
+  - [代码风格](#代码风格)
+    - [Python](#python)
+    - [C++ and CUDA](#c-and-cuda)
+
+感谢您对参与MMSelfsup贡献的兴趣。我们欢迎各种形式的贡献，包括但不限于：
```
Comment (fangyixiao18) [inline docs/zh_cn/notes/contribution_guides.md]:
```suggestion
感谢您对参与 MMSelfsup 贡献的兴趣。我们欢迎各种形式的贡献，包括但不限于：
```

**Review by fangyixiao18 — CHANGES_REQUESTED** [review]

**Comment by fangyixiao18** [comment]
As inline comments

File: docs/zh_cn/notes/contribution_guides.md (hunk 14 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 15 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 16 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

File: docs/zh_cn/notes/contribution_guides.md (hunk 17 above)
Comment (wangbo-zhao) [reply docs/zh_cn/notes/contribution_guides.md]: done

**Review by fangyixiao18 — APPROVED** [review]

**Pull request merged**